### PR TITLE
Added New Plaubooks and Module Fields

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
- Added IOC Hunt Playbooks
- Modified Threat Hunting Plaubooks (Moved configurable parameters to "Investigate Outbreak Alert" playbook)
- Added Signature Based ForetiSIEM and FortiAnalyzer fields in threat hunt rule module

### CODE REVIEW

File: modules/.DS_Store
Comment (cs-suyog-jinturkar) [inline modules/.DS_Store:1]: Please remove this file

File: modules/threat_hunt_rules/detail-layout.json [hunk 1]
```diff
@@ -181,34 +181,51 @@
                                                                                             {
                                                                                                 "fields": [
                                                                                                     {
-                                                                                                        "name": "qRadar",
+                                                                                                        "name": "sTHFortiSIEM",
```
Comment (cs-suyog-jinturkar) [inline modules/threat_hunt_rules/detail-layout.json:184]: sTHFortiSIEM? what is this name all about?

File: modules/threat_hunt_rules/detail-layout.json [hunk 2]
```diff
@@ -181,34 +181,51 @@
...
+                                                                                                        "readOnly": true,
+                                                                                                        "linky": true
                                                                                                     }
                                                                                                 ],
                                                                                                 "style": "col-lg-6"
                                                                                             },
                                                                                             {
                                                                                                 "fields": [
+                                                                                                    {
+                                                                                                        "name": "sTHFortiAnalyzer",
```
Comment (cs-suyog-jinturkar) [inline modules/threat_hunt_rules/detail-layout.json:213]: same as above?

File: playbooks/10 - SP - Outbreak Alert Management Framework/Create Ticket in Ticketing Tools.json
Comment (cs-suyog-jinturkar) [inline playbooks/10 - SP - Outbreak Alert Management Framework/Create Ticket in Ticketing Tools.json:1]: Sugestion: Create Ticket in ITSM tools

File: modules/threat_hunt_rules/detail-layout.json (hunk 1 above)
Comment (cs-sujeet-ahirrao) [reply modules/threat_hunt_rules/detail-layout.json:184]: Added tooltips

File: modules/threat_hunt_rules/detail-layout.json (hunk 2 above)
Comment (cs-sujeet-ahirrao) [reply modules/threat_hunt_rules/detail-layout.json:213]: Added tooltips

File: modules/threat_hunt_rules/detail-layout.json (hunk 1 above)
Comment (cs-suyog-jinturkar) [reply modules/threat_hunt_rules/detail-layout.json:184]: Thanks for the tooltip. 
1. Is STH is well known term in CyberSecurity?
2. Why STH only for FortiSIEM and why not for others like Splunk, etc.?
3. Tab already named as "Signature-Based Hunt Rules", why do we required STH again for FSIEM?

File: modules/outbreak_alerts/detail-layout.json [hunk 3]
```diff
@@ -111,27 +111,26 @@
...
                                                                                                         "name": "severity",
                                                                                                         "highlightMode": true,
-                                                                                                        "readOnly": true
-                                                                                                    },
-                                                                                                    {
-                                                                                                        "name": "status",
-                                                                                                        "highlightMode": true,
-                                                                                                        "readOnly": true
+                                                                                                        "readOnly": true,
+                                                                                                        "linky": true
```
Comment (cs-suyog-jinturkar) [inline modules/outbreak_alerts/detail-layout.json:121]: What is a use of this param? what happens if we mark it as false?

File: modules/threat_hunt_rules/detail-layout.json [hunk 4]
```diff
@@ -181,34 +181,51 @@
...
                                                                                                     {
-                                                                                                        "name": "qRadar",
+                                                                                                        "name": "sTHFortiSIEM",
                                                                                                         "highlightMode": true,
+                                                                                                        "linky": true,
                                                                                                         "readOnly": true
                                                                                                     },
+                                                                                                    {
+                                                                                                        "name": "qRadar",
+                                                                                                        "highlightMode": true,
```
Comment (cs-suyog-jinturkar) [inline modules/threat_hunt_rules/detail-layout.json:191]: what is a highlightMode for?

File: modules/threat_hunt_rules/detail-layout.json [hunk 5]
```diff
@@ -181,34 +181,51 @@
...
                                                                                                 ],
                                                                                                 "style": "col-lg-6"
                                                                                             },
                                                                                             {
                                                                                                 "fields": [
+                                                                                                    {
+                                                                                                        "name": "sTHFortiAnalyzer",
+                                                                                                        "highlightMode": true,
+                                                                                                        "linky": true,
+                                                                                                        "readOnly": false
```
Comment (cs-suyog-jinturkar) [inline modules/threat_hunt_rules/detail-layout.json]: why readonly = false for FAZ?

File: modules/threat_hunt_rules/detail-layout.json [hunk 6]
```diff
@@ -256,7 +273,7 @@
                                             }
                                         },
                                         {
-                                            "title": "Fortinet Fabric Hunting",
+                                            "title": "Fortinet Fabric Hunt Rules",
```
Comment (cs-suyog-jinturkar) [inline modules/threat_hunt_rules/detail-layout.json:276]: I can't see this title in UI. Can you please guide me?

File: playbooks/10 - SP - Outbreak Alert Management Framework/IOC Threat Hunt - FortiAnalyzer.json [hunk 7]
```diff
@@ -0,0 +1,392 @@
...
+        }
+    ],
+    "groups": [],
+    "priority": null,
+    "uuid": "ba5e8b33-545d-4206-b7ac-204eed2558bf",
+    "owners": [],
+    "isPrivate": false,
+    "deletedAt": null,
+    "recordTags": [
+        "PostCreate",
```
Comment (cs-suyog-jinturkar) [inline playbooks/10 - SP - Outbreak Alert Management Framework/IOC Threat Hunt - FortiAnalyzer.json]: this is reference pb, right?

File: playbooks/10 - SP - Outbreak Alert Management Framework/IOC Threat Hunt - FortiAnalyzer.json [hunk 8]
```diff
@@ -0,0 +1,392 @@
...
+            "stepType": "\/api\/3\/workflow_step_types\/04d0cf46-b6a8-42c4-8683-60a7eaa69e8f",
+            "group": null,
+            "uuid": "a428f75f-aa57-4f77-a599-6281ac713420"
+        },
+        {
+            "@type": "WorkflowStep",
+            "name": "Configuration",
+            "description": null,
+            "arguments": {
+                "ADOM": "{{vars.input.params['faz_ADOM']}}",
```
Comment (cs-suyog-jinturkar) [inline playbooks/10 - SP - Outbreak Alert Management Framework/IOC Threat Hunt - FortiAnalyzer.json:46]: Are we going to move these FAZ specific config into SP config wizard now?

File: playbooks/10 - SP - Outbreak Alert Management Framework/IOC Threat Hunt - FortiAnalyzer.json [hunk 9]
```diff
@@ -0,0 +1,392 @@
...
+            "status": null,
+            "top": "975",
+            "left": "475",
+            "stepType": "\/api\/3\/workflow_step_types\/0109f35d-090b-4a2b-bd8a-94cbc3508562",
+            "group": null,
+            "uuid": "39a910ae-89b0-4d34-b792-b6678e3ee879"
+        },
+        {
+            "@type": "WorkflowStep",
+            "name": "Found FortiSIEM Search Results",
```
Comment (cs-suyog-jinturkar) [inline playbooks/10 - SP - Outbreak Alert Management Framework/IOC Threat Hunt - FortiAnalyzer.json]: Suggestion: Found FortiSIEM Search Results ---> Found Hunt Results

Comment (cs-suyog-jinturkar) [reply playbooks/10 - SP - Outbreak Alert Management Framework/IOC Threat Hunt - FortiAnalyzer.json]: And why FSM if the PB is of FAZ?

Comment (cs-sujeet-ahirrao) [reply playbooks/10 - SP - Outbreak Alert Management Framework/IOC Threat Hunt - FortiAnalyzer.json]: updated step name

File: playbooks/10 - SP - Outbreak Alert Management Framework/IOC Threat Hunt - FortiAnalyzer.json (hunk 8 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/10 - SP - Outbreak Alert Management Framework/IOC Threat Hunt - FortiAnalyzer.json:46]: no we are taking all config for all integration from investigation playbook and pass to required hunt playbooks

File: playbooks/10 - SP - Outbreak Alert Management Framework/IOC Threat Hunt - FortiAnalyzer.json (hunk 7 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/10 - SP - Outbreak Alert Management Framework/IOC Threat Hunt - FortiAnalyzer.json]: updated tag

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**